components/OtherUserSpectrum.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <li>
+    <slot></slot>
+    <span class="other-user-spectrum-gradient" :class="gradientStyle">
+      <span class="other-user-circle" :style="{ left: positionStyle }"></span>
```

Comment:
I would bind the entire style object. see https://v2.vuejs.org/v2/guide/class-and-style.html?redirect=true#Object-Syntax-1

Reply:
If you're suggesting something like this:

`  data() {
    return {
      styleObject: {
        left: this.positionStyle()
      }      
    }
  },`
  
  this doesn't work because i can't call computed functions in data